food_ratings/frontend/views.py

```diff
@@ -56,74 +60,169 @@ def _entry(register, key):
     response = _get('%s/record/%s.json' % (url, key))
     return response.json()
 
-@frontend.route('/',  methods=['GET', 'POST'])
+def _entry_async(register, key, session):
+    register = _config(register, 'register')
+    url = "%s/record/%s.json" % (register, key)
+    current_app.logger.info("ASYNC GET: " + url)
```

Comment:
Please could this log response status code and text like the _entry() method does?